app/services/parkingPlacesService.js

```diff
@@ -1,9 +1,9 @@
-const { map, filter, flatten } = require('lodash');
+const { map, filter, flatten, isEmpty, findIndex } = require('lodash');
 
-const decorateParkingPlacesWithUser = (parkingPlaces, userName) =>
+const decorateParkingPlaces = (parkingPlaces, decorateProperties) =>
   map(parkingPlaces, ({ PlaceID }) => ({
     PlaceID,
-    Owner: userName,
+    ...decorateProperties,
```

Comment:
`decorate` means to do something.
`decorated` means that something is already decorated.

I think that your choice is `decoratedProperties` and `decoratedParkingPlaces`